closed/GensrcJ9JCL.gmk

```diff
@@ -47,26 +47,38 @@ ifeq (true,$(OPENJ9_ENABLE_OPENJDK_METHODHANDLES))
   JPP_TAGS += OPENJDK_METHODHANDLES
 endif # OPENJ9_ENABLE_OPENJDK_METHODHANDLES
 
-$(J9JCL_SOURCES_DONEFILE) : $(AllJclSource) $(AllDdrSource)
-	@$(ECHO) Building OpenJ9 Java Preprocessor
-	@$(MKDIR) -p $(J9TOOLS_DIR)
-	$(MAKE) $(MAKE_ARGS) -C $(OPENJ9_TOPDIR)/sourcetools -f buildj9tools.mk \
-		BOOT_JDK=$(BOOT_JDK) \
-		DEST_DIR=$(call FixPath,$(J9TOOLS_DIR)) \
-		JAVA_HOME=$(BOOT_JDK) \
-		preprocessor
-	@$(ECHO) Generating J9JCL sources
+# OpenJ9 CRIU only supports x86-64 Linux, so we only need to consider the unix sub-directory.
```

Comment:
Please remove `x86-64` from this comment: see [defaults.yml](https://github.com/eclipse-openj9/openj9/blob/master/buildenv/jenkins/variables/defaults.yml#L542-L546) - other architectures are supported.

Reply:
removed